src/main/kotlin/uk/gov/justice/digital/hmpps/welcometoprison/resources/PrisonResource.kt

```diff
@@ -116,12 +122,72 @@ class PrisonResource(
       ),
     ]
   )
-
+  @PreAuthorize("hasRole('ROLE_VIEW_ARRIVALS')")
   @GetMapping(value = ["/prison/{prisonId}"], produces = [MediaType.APPLICATION_JSON_VALUE])
   fun getPrison(
     @Schema(description = "Prison ID", example = "MDI", required = true)
     @PathVariable prisonId: String
   ): PrisonView = PrisonView(prisonService.getPrison(prisonId).prisonName)
+
+  @Operation(
+    summary = " Completes transfer-in for a specific prisoner",
+    description = "Completes transfer-in of prisoner, role required is TRANSFER_PRISONER",
+    security = [SecurityRequirement(name = "ROLE_TRANSFER_PRISONER", scopes = ["write"])],
+    responses = [
+      ApiResponse(
+        responseCode = "200",
+        description = "The prisoner transferred in",
+        content = [
+          Content(
+            mediaType = "application/json",
+            array = ArraySchema(schema = Schema(implementation = TransferIn::class))
+          )
+        ]
+      ),
+      ApiResponse(
+        responseCode = "401",
+        description = "Unauthorized to access this endpoint",
+        content = [Content(mediaType = "application/json", schema = Schema(implementation = ErrorResponse::class))]
+      ),
+      ApiResponse(
+        responseCode = "403",
+        description = "Incorrect permissions to post",
+        content = [Content(mediaType = "application/json", schema = Schema(implementation = ErrorResponse::class))]
+      ),
+      ApiResponse(
+        responseCode = "404",
+        description = "Prisoner number not found",
+        content = [Content(mediaType = "application/json", schema = Schema(implementation = ErrorResponse::class))]
+      ),
+      ApiResponse(
+        responseCode = "500",
+        description = "Unexpected error",
+        content = [
+          Content(
+            mediaType = "application/json",
+            schema = Schema(implementation = ErrorResponse::class)
+          )
+        ]
+      ),
+    ]
+  )
+
+  @PreAuthorize("hasRole('ROLE_TRANSFER_PRISONER') and hasAuthority('SCOPE_write')")
```

Comment:
Can we make this role the same as the others? the name is a bit rubbish but basically means we only need a single role to access this service

Reply:
done